packages/tools/devtools/devtools/src/index.ts

```diff
@@ -187,6 +192,14 @@ function mapContainerProps(
 	};
 }
 
+/**
+ * Attempts to retrieve the global Fluid Devtools instance, initializing it if it's not exists.
```

Comment:
We'll need to update these docs to reflect the new contract. Also, we shouldn't need to take in a `logger` parameter anymore.